### ISSUE TITLE
[Manager asks another Manager for help] don’t work

### ISSUE THREAD

**Issue by saleh-aleid** [zaiste/sprinkle#10]
Dear Zaiste,

Thank you very much for your great effort. I tried the (Manager asks another Manager for help) feature, but it didn't work for me.
example:
`var manager = use<MyManager>();`
error:
`The method 'use' isn't defined for the type 'HomeManager'.
Try correcting the name to the name of an existing method, or defining a method named 'use'.`

**Comment by zaiste** [comment]
@saleh-aleid Yes, I needed to (temporarily) disable that feature. Sorry for the confusion. I'll push a fix in the next few days.

**Comment by nejcsever** [comment]
@zaiste is this something that will be fixed? Is there any workaround in order to achieve that feature without `use` method?

**Comment by saleh-aleid** [comment]
@nejcsever you can use this:

`final managerName =  supervisor.summon<OtherManager>();`

**Comment by nejcsever** [comment]
@saleh-aleid Are you passing supervisor into constructor to other managers then?

**Comment by saleh-aleid** [comment]
@nejcsever no see this link:
[Automatic Provider Setup](https://github.com/zaiste/sprinkle#automatic-provider-setup)

**Comment by nejcsever** [comment]
Oh, ok got it it's imported as a constant. Thanks.
